packages/cli/src/commands/env/add.ts

```diff
@@ -219,17 +126,32 @@ export default async function add(
     }
   }
 
+  if (
+    !stdInput &&
+    !envGitBranch &&
+    envTargets.length === 1 &&
+    envTargets[0] === ProjectEnvTarget.Preview
+  ) {
+    const { inputValue } = await inquirer.prompt({
+      type: 'input',
+      name: 'inputValue',
+      message: `Add ${envName} to which Git Branch? (leave empty for all Prevew branches)?`,
+    });
+    envGitBranch = inputValue || '';
```

Comment:
Is the `|| ''` necessary? I figure the prompt would return empty string already.

Reply:
I have no idea since the prompt returns `any` so just doing this to be safe (we do it the same on the other prompts)